paymentintent/client.go

```diff
@@ -0,0 +1,129 @@
+// Package payment_intent provides the /payment_intents APIs
+package paymentintent
+
+import (
+	"net/http"
+
+	stripe "github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/form"
+)
+
+// Client is used to invoke /payment_intents APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// New POSTs a new payment_intent.
+// For more details see https://stripe.com/docs/api#create_payment_intent.
+func New(params *stripe.PaymentIntentParams) (*stripe.PaymentIntent, error) {
+	return getC().New(params)
+}
+
+func (c Client) New(params *stripe.PaymentIntentParams) (*stripe.PaymentIntent, error) {
+	payment_intent := &stripe.PaymentIntent{}
```

Comment:
And LOL, I didn't even notice these right away, but saw them in the Linter's error messages, but these of course should be `paymentIntent` instead of `payment_intent`.